api/flight_api.py

```diff
@@ -66,11 +66,11 @@ def cheapest_tickets(origin_country_code: str, destination_country_code: str, de
     return response
 
 
-def non_stop_tickets():
+def non_stop_tickets(origin_country_code: str, destination_country_code: str):
     url = "https://travelpayouts-travelpayouts-flight-data-v1.p.rapidapi.com/v1/prices/direct/"
 
-    querystring = {"destination": "ICN", "origin": "SIN",
-                   "return_date": "2023-09-19", "depart_date": "2023-09-02", "currency": "RUB"}
+    querystring = {"destination": destination_country_code, "origin": origin_country_code,
+                "currency": "USD"}
```

Comment:
meant to be USD? Cause upstairs was SGD. Consider putting currency as a variable in a constants file, or as a param to take in, so that users can customise which currency they wanna see